torch_xla/csrc/BUILD

```diff
@@ -211,18 +213,6 @@ cc_library(
     ],
 )
 
-ptxla_cc_library(
```

Comment:
What's this change for?

Reply:
Now, _aten_cpu_fallback.cpp_ needs functions from:

- _aten_xla_bridge.cpp_
- _xla_graph_executor.cpp_
- _dl_convertor.cpp_

So, I thought it would be easier to merge it into the main library.